website/src/index-page.tsx

```diff
@@ -1,119 +1,374 @@
-    image: {
-      src: uploadsImage,
-      placeholder: 'empty',
-      alt: 'Uploads',
-    },
-    flipped: true,
-    link: {
-      children: 'Learn more',
-      title: 'Learn more',
-      href: '/docs/features/file-uploads',
-    },
-  },
+const gradients: [string, string][] = [
+  ['#8b5cf6', '#6d28d9'], // violet
+  ['#06b6d4', '#0e7490'], // cyan
+  ['#f59e0b', '#d97706'], // amber
 ]
-import Link from 'next/link'
 
-export default function IndexPage() {
+function pickGradient(i: number) {
+  const gradient = gradients[i % gradients.length]
+  if (!gradient) {
+    throw new Error('No gradient found')
+  }
+  return gradient
+}
+
+export function IndexPage(): ReactElement {
+  return (
+    <div className="flex flex-col">
+      <FeatureWrapper>
+        <div className="w-full">
+          <div className="py-20 sm:py-24 lg:py-32 my-6">
+            <h1
+              className="max-w-screen-md mx-auto font-extrabold text-5xl sm:text-5xl lg:text-6xl text-center bg-gradient-to-r from-fuchsia-400 to-fuchsia-700 dark:from-fuchsia-400 dark:to-fuchsia-700 bg-clip-text text-transparent"
+              style={{
+                // TODO: default line height is 1, but that cuts of the "g" in the title
+                lineHeight: 1.2,
+              }}
+            >
+              GraphQL Yoga
+            </h1>
+            <p className="max-w-screen-sm mx-auto mt-6 text-2xl text-gray-600 text-center dark:text-gray-400">
+              The fully-featured GraphQL Server with focus on easy setup,
+              performance and great developer experience.
+            </p>
+            <div className="mt-10 flex flex-col sm:flex-row items-center justify-center gap-4">
+              <ButtonLink href="/docs">Documentation</ButtonLink>
+              <ButtonLink className="hidden lg:block" href="/tutorial/basic">
+                Tutorial
+              </ButtonLink>
+              <ButtonLink
+                className="flex flex-row gap-2 items-center"
+                href="https://github.com/dotansimha/graphql-yoga"
+              >
+                <FiGithub /> GitHub
+              </ButtonLink>
+            </div>
+          </div>
+        </div>
+      </FeatureWrapper>
+      <Feature
+        title="GraphQL over HTTP compliant"
+        description={
+          <>
+            following the&nbsp;
+            <TextLink
+              href="https://graphql.github.io/graphql-over-http"
+              className="text-primary-500"
+            >
+              GraphQL over HTTP specification
+            </TextLink>
+          </>
+        }
+        gradient={0}
+      >
+        <div className="flex flex-col md:flex-row gap-12 justify-between">
+          <FeatureHighlights
+            highlights={[
+              {
+                title: 'Passes all audits',
+                description: (
+                  <>
+                    Tested using the graphql-http library.&nbsp;
```

Comment:
Why to use nbsp here and below? Why non breaking space here is intended?

Reply:
Because there's no space otherwise. You recommend using `{'  '}` instead>

Reply:
If it not add a space, add space like this
```suggestion
                    Tested using the graphql-http library.{' '}
```

Reply:
Yes, {' '} is better approach, replace all nbsp with it